### ISSUE TITLE
Update docs to state that it needs Node 16+ (WAS: Node 14+ is now required after commander bump)

### ISSUE THREAD

**Issue by jaredkirkley** [knex/knex#5676]
Upgrading to Commander 10.0.0 requires Node 14+, the docs should be updated to indicate this project now requires Node 14+, or this requirement should be addressed.

https://github.com/tj/commander.js/releases/tag/v10.0.0

**Comment by rubengmurray** [comment]
Hi @jaredkirkley. The required version is currently documented in `package.json`, at least in the latest published version of `knex`, via the "engines.node" value (currently `>=16`).

I do agree that this could be also made clearer in the installation guide https://knexjs.org/guide/#node-js though!

cc @mercmobily

**Comment by jaredkirkley** [comment]
> Hi [@jaredkirkley](https://github.com/jaredkirkley). The required version is currently documented in `package.json`, at least in the latest published version of `knex`, via the "engines.node" value (currently `>=16`).
> 
> I do agree that this could be also made clearer in the installation guide https://knexjs.org/guide/#node-js though!
> 
> cc [@mercmobily](https://github.com/mercmobily)

Yeah it looks like the package.json was updated a little less than a month after I posted this 2 years ago. Seems fine now

**Comment by mercmobily** [comment]
Repoening so that we update the docs.
